mainframer.sh

```diff
@@ -70,6 +70,22 @@ if [ -z "$REMOTE_COMMAND" ]; then
 	exit 1
 fi
 
+function formatTime {
+    local time=$1
+
+    local hours=$((time/3600))
+    local minutes=$(((time % 3600) / 60))
+    local seconds=$((time % 60))
+
+    if [ "$hours" -eq "1" ]; then HOURS_LABEL="hour"; else HOURS_LABEL="hours"; fi
+    if [ "$minutes" -eq "1" ]; then MINUTES_LABEL="minute"; else MINUTES_LABEL="minutes"; fi
+    if [ "$seconds" -eq "1" ]; then SECONDS_LABEL="second"; else SECONDS_LABEL="seconds"; fi
+
+    (( hours > 0 )) && printf "%d $HOURS_LABEL " ${hours}
+    (( minutes > 0 )) && printf "%d $MINUTES_LABEL " ${minutes}
+    (( seconds > 0 )) && printf "%d $SECONDS_LABEL \n" ${seconds}
```

Comment:
If you take a look at Travis CI output carefully, you can see this.

```
Sync local → remote machine...
Sync done: took 

Executing command on remote machine…
```

In other words, seconds count is `0`, so we output nothing. You add `seconds >= 0`, this should do the trick.

BTW, not sure if it is possible, but can we test this? CC @artem-zinnatullin